posthog/hogql/modifiers.py

```diff
@@ -50,6 +50,9 @@ def create_default_modifiers_for_team(
     else:
         modifiers = modifiers.model_copy()
 
+    if modifiers.useMaterializedViews is None:
+        modifiers.useMaterializedViews = True
+
     if isinstance(team.modifiers, dict):
```

Comment:
@EDsCODE Just curious if this is what you meant by defaulting and if it's appropriate to do this.